### PR TITLE
Custom Keycloak login theme

### PR DESCRIPTION
**Pull request by dnlkoch**
This adds a custom login theme for Keycloak based on the [official guide](https://www.keycloak.org/docs/latest/server_development/#creating-a-theme).

### CODE REVIEW

**Comment by faouziH21** [comment]
The Login theme should look like this


image

**Comment by faouziH21** [comment]
Final version:

image

File: shogun-keycloak/themes/shogun/login/resources/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
+.login-pf body {
+
+  background: white;
+  margin-left: auto;
+  margin-right: auto;
+  font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```
Comment (dnlkoch) [inline shogun-keycloak/themes/shogun/login/resources/css/styles.css:6]: Is this font also used in the other forms, e.g. the password reset one?

File: shogun-keycloak/themes/shogun/login/resources/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  text-align: center;
+  padding: 2%;
+
+}
+
+.pf-c-button.pf-m-primary {
+
+  background-color: #0b5ed7;
+
+}
```
Comment (dnlkoch) [inline shogun-keycloak/themes/shogun/login/resources/css/styles.css:101]: Very minor, but the blank lines in the selector groups can be removed.

File: shogun-keycloak/themes/shogun/login/resources/css/styles.css (hunk 1 above)
Comment (faouziH21) [reply shogun-keycloak/themes/shogun/login/resources/css/styles.css:6]: yes

File: shogun-keycloak/themes/shogun/login/resources/css/styles.css (hunk 2 above)
Comment (faouziH21) [reply shogun-keycloak/themes/shogun/login/resources/css/styles.css:74]: done

**Comment by dnlkoch** [comment]
Thanks @faouziH21!

**Pull request merged**